sdk/cognitiveservices/Language.TextAnalytics/src/README.md

```diff
@@ -14,8 +13,5 @@
 * .NET Framework 4.5.2
 * Netstandard 1.4, based on the NetCore framework
 
-## Build Instructions:
```

Comment:
I removed this section as build instructions are not working. Also, looking around at other services, they don't seem to have this information